ui/src/components/common/AppItem.jsx

```diff
@@ -20,14 +20,29 @@ class AppItem extends PureComponent {
         }}
       />
     );
-    return (
+    const ftm_message = (
+      <FormattedMessage
+        id="footer.aleph"
+        defaultMessage="followthemoney {version}"
```

Comment:
This is absolutely nitpicking, but I think it would make sense if we tried to get a little more consistent about the casing we use for FtM. So far, we tend to use *FollowTheMoney* in the docs when talking about the data model and/or project as a whole.

```suggestion
        defaultMessage="FollowTheMoney {version}"
```